src/page/MoviePage.jsx

```diff
@@ -42,10 +54,10 @@ export default class MoviePage extends React.Component {
 
   searchMovies(page) {
     const { searchQuery, getActionDispatcher } = this.props;
-    return getSearchMovies(searchQuery, page)
+    return getMovies({ searchQuery, page })
       .then(({ movies, totalResults }) => {
         getActionDispatcher({
```

Comment:
Don't create actions here. This is antipattern. Use action creators please.
https://redux.js.org/basics/actions